src/core/jsonschema/include/sourcemeta/core/jsonschema.h

```diff
@@ -5,20 +5,15 @@
 #include <sourcemeta/core/jsonschema_export.h>
 #endif
 
-#include <sourcemeta/core/json.h>
+#include <sourcemeta/core/json.h> // NOLINT(misc-include-cleaner) Public header needs it
```

Comment:
@jviotti Let me know if this is not what you meant

Reply:
I mean that for the public header of each module, we ignore its private includes. For example, in `jsonschema.h`, I presume you can do this:

```c++
// NOLINTBEGIN(misc-include-cleaner)

#include <sourcemeta/core/jsonschema_error.h>
#include <sourcemeta/core/jsonschema_frame.h>
#include <sourcemeta/core/jsonschema_resolver.h>
#include <sourcemeta/core/jsonschema_transform.h>
#include <sourcemeta/core/jsonschema_types.h>
#include <sourcemeta/core/jsonschema_walker.h>

// NOLINTEND(misc-include-cleaner)
```

Essentially, for a header `X.h`, then don't clean its `X_<private>.h` includes

Reply:
And we can do the same for every public header of every module

Reply:
Sure for every public header of each module.